### PR TITLE
Local groups index page

### PR DESCRIPTION
**Pull request by franei**
What's in this PR:
- A template for an index page that lists all local groups, with links to their individual pages
- Custom fields for a few settings and text snippets
- CSS

Happy to implement any feedback :)

#### Custom fields
acf-index-page

#### Index page
lg-index

### CODE REVIEW

File: web/app/themes/xrnl/local_group-archive.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  <div class="col-11 col-sm-10 col-lg-8 mx-auto">
+
+    <div class="index-container">
+      <div class="select-region">
+        <button type="button" name="all-regions" class="active" id="show-all-btn"><?php _e('All regions', 'theme-xrnl'); ?></button>
+        <button type="button" name="noord-west" class="region-select-btn">Noord-West</button>
+        <button type="button" name="noord-oost" class="region-select-btn">Noord-Oost</button>
+        <button type="button" name="midden" class="region-select-btn">Midden</button>
+        <button type="button" name="zuid-west" class="region-select-btn">Zuid-West</button>
+        <button type="button" name="zuid-oost" class="region-select-btn">Zuid-Oost</button>
```
Comment (SamLubbers) [inline web/app/themes/xrnl/local_group-archive.php:43]: do we also want to translate the regions?

**Review by SamLubbers — APPROVED** [review]
@franei thank you for getting this done :raised_hands: :raised_hands: This was one of the key missing features on the website. 

One thing I would add to the design is showing the XR logo of each local group, as most local groups have a very nice logo characteristic of their city. If you also like the idea and think it will be very easy you can add it straight away, otherwise you can add it to the Trello as a future idea and merge this pull request :ok_hand:

File: web/app/themes/xrnl/local_group-archive.php (hunk 1 above)
Comment (franei) [reply web/app/themes/xrnl/local_group-archive.php:43]: ah yes I wasn't sure about this.. to me it often looks weird when place names get translated, but I guess something like "Midden" is borderline.. it's as much a description as a name. Let's make them translatable then we can always change it easily in the String translation settings :)

**Comment by franei** [comment]
I really like the idea of adding the logos, it will make the page look less boring. If it's ok with you i won't do it right now, because then I can help out with the Events page first (also I think it's only a handful of groups that have a logo uploaded at the moment).

**Comment by SamLubbers** [comment]
@franei yes that's cool 👌

**Pull request merged**